uw-frame-components/js/frame-config.js

```diff
@@ -348,7 +348,7 @@ define(['angular'], function(angular) {
             }
           },
           {
-            "name" : "uw-oskhosh",
+            "name" : "uw-oshkosh",
```

Comment:
If you change this name  also update the theme version, and change the theme file name in `uw-frame-components/css/themes`